bedrock/firefox/templates/firefox/accounts.html

```diff
@@ -38,7 +38,7 @@ <h1>{{_('Sign up, sign in, stay in sync.')}}</h1>
       <div class="fxa-cta show-fxa-supported-signed-in show-fxa-supported-signed-out">
         <p class="show-fxa-supported-signed-in signed-in">{{_('You’re signed in and ready to start using Sync.')}}</p>
         <div class="show-fxa-supported-signed-out">
-          {{ fxa_email_form(entrypoint='accounts-page', utm_source='accounts-page', utm_params={'campaign': 'fxa-embedded-form', 'content': 'accounts-page', 'medium': 'referral'}) }}
+          {{ fxa_email_form(entrypoint='mozilla.org-accounts', utm_source='accounts-page', utm_params={'campaign': 'fxa-embedded-form', 'content': 'accounts-page', 'medium': 'referral'}) }}
```

Comment:
Should this be `mozilla.org-accounts_page` to match the other template, or is it better being something unique?

Reply:
I will update it to match the other template, I'm not sure if they need to be unique.